Code-Hammers/code-hammers
# [CHE-190] Add User Feedback

Thank you for your contribution to Code Hammers's repository! Before merging, please fill out this brief form.

### Description

-Provides feedback messages for users on the registration page related to invalid or expired tokens or mismatched passwords. 


### Jira Task

[CHE-190](https://codehammers.atlassian.net/browse/CHE-190?atlOrigin=eyJpIjoiMDNhNTAzNGRmMTIwNDY4MWJiZThiYTZlOGYyZDI0NDEiLCJwIjoiaiJ9)

### Testing Instructions

Go to /registration endpoint and put in:
- a bad token or no token and confirm that appropriate error messages are showing up
- mismatched passwords

### Checklist

#### All Team Members

- [x ] I filled out the **Description**, **Jira Task**, and **Testing Instructions** sections above.
- [x ] I did a quick check to make sure my code changes follow the recomended style guide.

### Additional Notes, Images, etc.




[CHE-190]: https://codehammers.atlassian.net/browse/CHE-190?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
